package.json

```diff
@@ -36,6 +36,8 @@
     "debug": "node --inspect-brk ./lighthouse-cli/index.js",
     "start": "yarn build-report --standalone && node ./lighthouse-cli/index.js",
     "jest": "node --experimental-vm-modules ./node_modules/jest/bin/jest.js",
+    "____comment": "echo https://github.com/cucumber/cucumber-js/issues/1844",
+    "mocha": "cp node_modules/mocha/bin/mocha node_modules/mocha/bin/mocha.js && node --loader=testdouble node_modules/mocha/bin/mocha.js --require=lighthouse-core/test/mocha-setup.js",
```

Comment:
pop `--parallel` in there?   https://mochajs.org/#parallel-tests

however note this bit https://mochajs.org/#exclusive-tests-are-disallowed

on my machine the cli tests go from 10s to 6s in parallel. not a huge deal. probably something more important to consider for core's unit tests.